### ISSUE TITLE
[eslint-plugin-putout]: `--fix` breaks enums.

### ISSUE THREAD

**Issue by ElPrudi** [coderaiser/putout#169]
Every time this eslint plugin fixes a rule, it formats the file. This formatting breaks enums:

```ts
export const enum ABC {
  A,
  B,
  C,
  D,
  E
}

const x = typeof 5 === 'number' // example rule to trigger the putout formatter
```

turns into:

```ts
const isNumber = (a): a is number => typeof a === 'number'
export const enum ABC {
  A =,
  B =,
  C =,
  D =,
  E =
}

const x = isNumber(5) // example rule to trigger the putout formatter
```

**Comment by coderaiser** [comment]
Fixed in [`@putout/printer@2.87`](https://github.com/putoutjs/printer/releases/tag/v2.87.0) 🎉. Please re-install 🐊.

Is it works for you?

**Comment by ElPrudi** [comment]
Yes, now it doesn't try to change it anymore. Thank you very much :)
